src/core/constants.js

```diff
@@ -336,3 +337,7 @@ export type PromotedCategoryType =
   | typeof SPOTLIGHT
   | typeof STRATEGIC
   | typeof VERIFIED;
+
+export const PROMOTED_ADDON_EVENT_URL = `${config.get('apiPath')}${config.get(
```

Comment:
(conceptually), can you have config in constants?  Isn't it like a non-constant "constant" at that point?

Reply:
I think we shouldn't do that.

Reply:
Yeah, I wasn't so sure of this myself. I guess I'll move that into `tracking.js`.

Reply:
Why not keeping it in the component?